Task/43 - [BUG] Issue with displaying LaunchScreenActivity on diffrent screens

Zalezny opened the pull request:
**1. Created new 3 layouts for landscape, smaller screens and tablets screens for LaunchScreenActivity, Examples:**

![small phone](https://user-images.githubusercontent.com/65240240/220162378-9d9b55fb-f1f8-4278-8e77-a52bb8f81d96.PNG)

/\ small screen

![tablet](https://user-images.githubusercontent.com/65240240/220161401-adbe761e-611c-4760-8075-bc8968c58540.PNG)

/\ tablet

![Pixel 6 landscape](https://user-images.githubusercontent.com/65240240/220161440-7d091826-a823-4b70-bc7b-39949f46f870.PNG)

/\ Pixel 6 landscape

![Pixel 6 portrait](https://user-images.githubusercontent.com/65240240/220161448-34dd28a0-a0ba-4d7f-8bee-66732819898c.PNG)

/\ Pixel 6 portrait

**2. Created missing annotaion about LaunchScreenActivity in AndroidManifest.**

przemek19980102 reviewed (changes requested):
<img width="357" alt="image" src="https://user-images.githubusercontent.com/32803402/221347646-5f81a51a-4db3-4336-8b2e-3b1ed6fa9f8e.png">
That's how it looks on tablets

+ could you try to use max height attributes on imageView in order to get rid of this many layout variations? In order to complete the task we can have only one layout. Ask me for help if you have any questions :)

Zalezny commented:
Added changes: one xml file, added guidelines etc.
 

### Small screen:

![image](https://user-images.githubusercontent.com/65240240/221427214-4e44e8b3-bbb0-4fe3-8164-ab551f2840fb.png)
![image](https://user-images.githubusercontent.com/65240240/221427223-92c5142a-258d-425b-a303-8d0205e2fad2.png)

### Tablet:

![image](https://user-images.githubusercontent.com/65240240/221427257-c2e7dc9d-e125-443c-9407-68102aa0a70e.png)
![image](https://user-images.githubusercontent.com/65240240/221427267-8301684d-385c-4667-b9f8-d191f05b4be3.png)

### Phone (Pixel 6):
![image](https://user-images.githubusercontent.com/65240240/221427309-dc511de2-3d9e-474a-ac93-d1c784380868.png)
![image](https://user-images.githubusercontent.com/65240240/221427317-f646d45e-b4c4-49b2-b55d-c3a820b2787f.png)

msarelo commented on app/src/main/res/layout/activity_launch_screen.xml:
```diff
@@ -82,13 +83,45 @@
         android:enabled="true"
         android:gravity="center"
-        android:padding="14dp"
+        android:padding="@dimen/size_2x"
```
In zepplin there is 14dp:
![image](https://user-images.githubusercontent.com/3758285/222990614-ccbf64e5-83c6-4fed-869b-1ff0989c9230.png)

On my phone text is cut off from the bottom:
![image](https://user-images.githubusercontent.com/3758285/222990659-7beb4f46-a1da-48b0-884e-98f3558718ec.png)

przemek19980102 commented on app/src/main/res/layout/fragment_launch_screen.xml:
```diff
@@ -13,15 +13,14 @@
         android:letterSpacing="0.04"
         android:text="@string/launch_screen_txt_sign_in"
         android:textColor="@color/s_60"
         android:textSize="14sp"
```
Don't we have a style for the text already? You shouldn't need to put fontFamily, letterSpacing etc.
